src/model_parser.h

```diff
@@ -68,9 +68,8 @@ class ModelParser {
     ENSEMBLE_SEQUENCE
   };
 
-  explicit ModelParser(cb::BackendKind backend_kind)
```

Comment:
context: removed the parameter because it was being used nowhere in the model parser.